### PR TITLE
Create home page

### PR DESCRIPTION
**Pull request by EnriqueGambra**
@DerrickPersaud @brianseidl -- so Derrick can see the PR

- Created the default home page component
- Modified some CSS classes to now allow the entire page to transition when hiding/showing the sidebar

Home Page Screenshots: 
home-page-1
home-page-2

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,9 +21,8 @@ class App extends React.Component {
     return (
       <div>
         {/* TODO: Put signout feature in layout component */}
-        <AmplifySignOut />
+        {/* <AmplifySignOut /> */}
```
Comment (EnriqueGambra) [inline src/App.js:24]: Will have to move the AmplifySignOut component to display when we select the user profile icon in a dialog potentially

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -23,14 +27,19 @@ class Layout extends Component {
...
       <React.Fragment>
-        <div className="App wrapper">
+        <div className="App">
           <SideBar toggle={this.toggle} isOpen={this.state.isOpen} />
-          <NavBarContainer
-            toggle={this.toggle}
-            isOpen={this.state.isOpen}
-            showDialog={this.handleShowDialog}
-          />
+          <Container
```
Comment (EnriqueGambra) [inline src/components/Layout/Layout.js:32]: @DerrickPersaud I deleted the NavBarContainer component. We need this Container component wrapped around the entire body of the app in order for the app to respond appropriately when toggling the sidebar on and off. This change does that

File: src/pages/Home/Home.js [hunk 3]
```diff
@@ -1,16 +1,73 @@
...
+          </div>
+          <Cards cards={cardItems} />
+        </div>
+      </React.Fragment>
     );
   }
+
+  /**
+   * Notes:
+   * - Check for if a user has any boards... make API call to receive the data...
```
Comment (EnriqueGambra) [inline src/pages/Home/Home.js:68]: Just left this in here for now... still thinking through some things

File: src/App.js (hunk 1 above)
Comment (brianseidl) [reply src/App.js:24]: This is fine.  No need to have the sign out button here at this time

**Review by brianseidl — APPROVED** [review]

**Pull request merged**